discovery-provider/src/api/v1/metrics.py

```diff
@@ -92,6 +97,16 @@ def get(self):
 metrics_app_name_list_parser.add_argument('offset', required=False, type=int)
 metrics_app_name_list_parser.add_argument('include_unknown', required=False, type=inputs.boolean)
 
+@ns.route("/routes/trailing/month", doc=False)
+class RouteMetricsTrailingMonth(Resource):
+    @ns.marshal_with(route_metrics_trailing_month_response)
+    @cache(ttl_sec=3 * 60 * 60)
```

Comment:
Aren't these matviews refreshed in one hour intervals? For the cache it would probably make sense to invalidate like 30 mins?

Reply:
100%, didn't look closely at this value. 30 min sounds right